app/models/compute_resources/foreman/model/libvirt.rb

```diff
@@ -86,6 +86,19 @@ def storage_pools
       client.pools rescue []
     end
 
+    def bridges
+      # before ruby-libvirt fixes https://bugzilla.redhat.com/show_bug.cgi?id=1317909 we have to use raw XML to get type
+      bridges = client.client.list_all_interfaces.select do |libvirt_interface|
+        type_match = libvirt_interface.xml_desc.match /<interface.*type='([a-z]+)'/
```

Comment:
I see two potential issues here: if there is another `type` attribute in the xml, such as `"<interface type='a'><subnet type='b'>"`, this regular expression would return `b` instead of `a`. It should be solved by `.*?` to not use the greedy variant. The second problem is with the `'`, which might change in any future release of libvirt.